lab-justin/server.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+
+const http = require('http');
+const port = process.env.PORT || 3000;
+const parseUrl = require('url').parse;
+const parseQuery = require('querystring').parse;
```

Comment:
the url module will already parse query strings for you--you don't need to require in another package.